wpilibc/src/main/native/cppcs/RobotBase.cpp

```diff
@@ -22,6 +24,22 @@
 #include "frc/livewindow/LiveWindow.h"
 #include "frc/smartdashboard/SmartDashboard.h"
 
+namespace frc{
+class CameraServerShared {
```

Comment:
Doesn't doing this violate ODR?

Reply:
It's the same definition as in the CameraServer library, and isn't this exactly what #include does?

Reply:
I don't see why we can't just have a header-only dependency against cameraserver here.

Reply:
Gradle actually doesn't have a way to do header only, without literally just adding that as an extra include directory to an external folder.

Reply:
I would prefer adding the extra include directory.  I would be concerned about these two copies of this definition getting out of sync.  The part about ODR I don't know is the default virtual destructor, but practically speaking it's probably fine.

Reply:
I'll do the extra include directory then. However about ODR wouldn't the linker have to know how to handle that, since the header is likely to be included in multiple TU's?

Reply:
ODR is a language thing.  The language really doesn't know about DLLs or linkers...

Reply:
If it was an odr issue, any class with a default virtual destructor included via a header would be an odr violation then. But I'm now including the header.